lib/GLPI/Agent/Task/Inventory/Linux/Distro/OSRelease.pm

```diff
@@ -23,6 +23,14 @@ sub doInventory {
     _fixDebianOS(file => '/etc/debian_version', os => $os)
         if canRead('/etc/debian_version');
 
+    # Handle Astra Linux version information
+    _fixAstraOS(file => '/etc/astra/build_version', os => $os)
+        if canRead('/etc/astra/build_version');
+
+    # Handle Astra Linux license information
+    _fixAstraLicense(file => '/etc/astra_license', os => $os)
+        if canRead('/etc/astra_license');
+
```

Comment:
Here, can we assume `/etc/astra_license` should exist when `/etc/astra/build_version` has been found ?
If yes, you shold probably move `_fixAstraLicense()` under `_fixAstraOS()` function. But this is not a big deal.

Can you also provide `/etc/astra/build_version` and `/etc/astra_license` contents to add tests in `t/tasks/inventory/linux/distro.t` ?

Or you can merge `_fixAstraLicense()` under `_fixAstraOS()` and only call `_fixAstraOS()` if `/etc/astra/build_version` and `/etc/astra_license` can be read.